docs/faq.md

```diff
@@ -756,3 +771,48 @@ On examining the container structure with [Dive](https://github.com/wagoodman/di
 The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a library file by placing it in `src/main/jib/lib/libfoo.so`. This would normally cause the `libfoo.so` to be installed in the image as `/lib/libfoo.so`. But `/lib` and `/lib64` in the user's base image were symbolic links. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib` as a new directory. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
 
 Solution: The user installed the file in a different location.
+
+## Jib CLI 
+ 
+### How does the `jar` command support Standard JARs?
+
+A standard JAR can be containerized by the `jar` command in two modes, exploded or packaged. 
+
+#### Exploded Mode (Recommended)
+Achieved by calling `jib jar --target ${TARGET_REGISTRY} ${JAR_NAME}.jar`
```

Comment:
I think we can make the contents less verbose overall? Using less words. For example, instead of "It is the default mode for containerizing a JAR", just "The default mode."

Also check out https://github.com/GoogleContainerTools/jib/blob/02c9b5ce3e40c80f93ee900e4903c6d2cdc71cee/docs/faq.md#how-are-jib-applications-layered I think the contents have too much details? I think we don't need to mention `MANIFEST.MF`.

For example,

Dependencies Layer: other dependency JARs (if exist)
Snapshot-Dependencies Layer: snapshot dependencies (if exist)

Reply:
You're right. I cleaned it up a bit to make it less verbose.